checkstyle.xml

```diff
@@ -126,7 +126,9 @@
         </module>
         <module name="IllegalInstantiation"/>
         <module name="InnerAssignment"/>
-        <module name="MagicNumber"/>
+        <module name="MagicNumber">
```

Comment:
looks like it should be independent commit because it doesn't have any connections with Swagger.
Am I right?